### DISCUSSION TITLE
Contributions

### DISCUSSION THREAD

**Discussion by Zarinia** [salman0ansari/whatsapp-api-nodejs#203] [Q&A] [answered]
Hello,
How can I become one of your contributors?

**Comment by salman0ansari** [comment, marked as answer]
- refactor code
- add new endpoints
- find bugs
- help fix issues

**Reply by salman0ansari** [reply]
https://github.com/salman0ansari/whatsapp-api-nodejs/blob/main/CONTRIBUTING.md
